models/classes/LtiLink.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2017 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ *
+ */
+
+namespace oat\taoLti\models\classes;
+
+interface LtiLink
+{
+    const CLASS_INCOMING_URI = 'http://www.tao.lu/Ontologies/TAOLTI.rdf#LtiIncomingLink';
```

Comment:
Are you sure that you need so many interfaces? maybe it could be only one to describe all rdf constants?

Reply:
these are already defined in OntologyLink

Reply:
Except the PROPERTY_LAUNCH_URL I will move this as well to OntologyLink and remove the interface